cpuid/src/bit_helper.rs

```diff
@@ -32,6 +32,8 @@ pub struct BitRange {
     pub lsb_index: u32,
 }
 
+/// Trait containing helper methods for BitRange
```

Comment:
Can you give a bit of insight into the **why** we have this Trait?

Reply:
Done